content/en/blog/2019/istioctl-describe/index.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "'istioctl describe' in Istio 1.3"
+description: Introducing a command-line utility to find configuration affecting pod traffic.
+publishdate: 2019-09-20
+attribution: Ed Snible (IBM)
+keywords: [traffic-management, istioctl, debugging]
+---
+
+Istio 1.3 includes an experimental command-line tool designed to help find and
```

Comment:
Please be specific. This is not a new command line tool. This is a new subcommand for `istioctl` correct? If that's the case then please refer to it as such everywhere on the page. For example:
```suggestion
In Istio 1.3, we included the experimental `describe pod` subcommand for `istioctl`. We designed this tool to help find and
```